### PR TITLE
Add SkipAnalyzers=true to component declaration compilation

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/dotnet/aspnetcore/issues/25365

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -329,9 +325,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>4be47e467013f8a07a1ed7b6e49e39c8150bde54</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.8.0-2.20403.2" Pinned="true">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.8.0-3.20458.6">
```
Comment (pranavkm) [inline eng/Version.Details.xml:328]:
```suggestion
    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.8.0-3.20458.6" Pinned="true">
```

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -9,10 +9,6 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Components.WebAssembly.Runtime" Version="3.2.0">
```
Comment (pranavkm) [inline eng/Version.Details.xml:12]: This package is gone. Happened to notice this when editing this file

Comment (dougbu) [reply eng/Version.Details.xml:12]: Please remove https://github.com/dotnet/aspnetcore/blob/f0a5ad6d68524860f40d305fd862dd86e6b551d3/eng/Versions.props#L135-L136 too

**Review by dougbu — APPROVED** [review]

File: eng/Version.Details.xml (hunk 2 above)
Comment (pranavkm) [reply eng/Version.Details.xml:12]: Done

**Comment by pranavkm** [comment]
@Pilchie one more for rc2.

**Comment by Pilchie** [comment]
Approved for .NET Core 5.0 RC2 pending CI Completion.

**Pull request merged**